packages/cli/plopfile.js

```diff
@@ -0,0 +1,46 @@
+const promptDirectory = require('inquirer-directory');
+
+module.exports = function(plop) {
+  plop.setPrompt('directory', promptDirectory);
+
+  plop.setGenerator('ngxs', {
+    description: 'Create new store',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'Write store name, please...'
+      },
+      {
+        type: 'list',
+        name: 'storeType',
+        choices: ['Empty Store', 'Filled Store'],
```

Comment:
At first I was confused what empty vs filled is. Would be nice to help clarify that.

Reply:
This means that the user will have a choice:
- empty store (skeleton)
- A story filled with some data to learn how it works, for example

In the future, we can add a mini template for authorization, for example, both on the site. This will help a person who just starts using ngxs to look at it right away in the work, rather than studying the documentation. Perhaps it will be useful.

Reply:
@amcdnl If you want skip generate Examples any time, we can customize NGXS CLI in package.json later

Add to package.json

```json5
{
 ...
 "ngxsCli": {
   "skipExamples": true
 }
...
}
```

Schema CLI conf

```ts
interface NgxsCliImpl {
  skipExamples: boolean;
  defaultChooseDir: string;
}
```

Reply:
@amcdnl remove it